test/unit/manipulation.js

```diff
@@ -3099,3 +3099,55 @@ testIframe(
 		} );
 	}
 );
+
+// Custom event handler test case
+
+QUnit.test( "should handle custom '_se-custom-destroy' event correctly", function( assert ) {
+
+	// Define the expected number of assertions
+
+	assert.expect( 4 );
```

Comment:
There should be no empty line between a comment & code this comment describes.

This applies to many places in this PR; please analyze all the comments & fix as appropriate.

Reply:
Done